t/ingredients.t

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/perl -w
 
-use Modern::Perl '2017';
+use Modern::Perl '2014';
```

Comment:
Why this change ?

Reply:
oh this I was trying to run docker build and I made some changes since I was getting errors, I forgot to undo the change. We can keep 2017.

Reply:
It should work with 2017, but I don't know why the Perl unit tests are failing when I change it to 2017.

Reply:
@borrokk I think you forgot to include expected_test_results/ingredients/en-emulsifier-synonyms.json

Look at [log of last test run](https://pipelines.actions.githubusercontent.com/serviceHosts/2177ada6-e616-4c97-899b-f91e94e24e4c/_apis/pipelines/1/runs/63725/signedlogcontent/8?urlExpires=2022-04-25T13%3A44%3A23.9964973Z&urlSigningMethod=HMACV1&urlSignature=eC2X6WTBGvpvnNg1HzAPgBYGDmeIOUYS1%2FTF%2FORaCfM%3D)

> 2022-04-23T07:09:56.3850948Z #   Failed test 'could not load expected_test_results/ingredients/en-emulsifier-synonyms.json'
2022-04-23T07:09:56.3851403Z #   at t/ingredients.t line 455.

Reply:
Yes, I will look into this and make the changes.

Reply:
I added the json file but still getting errors. May I know what changes I need to make?